test_scripts/Polices/External_UCS/011_ATF_P_TC_SnapshotPT_without_disallowed_by_external_consent_entities_off_struct.lua

```diff
@@ -0,0 +1,93 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: SnapshotPT without "disallowed_by_external_consent_entities_off" struct
+--
+-- Description:
+-- In case:
+-- SDL Policies database omits “disallowed_by_external_consent_entities_off:
+-- [entityType: <Integer>, entityId: <Integer>]” struct -> in "<functional grouping>"
+-- -> from "functional_groupings" section,
+-- and SDL is triggered to create a SnapshotPolicyTable
+-- SDL must:
+-- omit this "disallowed_by_external_consent_entities_off: [entityType: <Integer>, entityId: <Integer>]" field
+-- in the corresponding "<functional grouping>" -> in the SnapshotPolicyTable.
+--
+-- Preconditions:
+-- 1. Start SDL (make sure 'disallowed_by_external_consent_entities_off' section is omitted in PreloadedPT)
```

Comment:
@istoimenova, By default this sections is not defined in PreloadedPT,
But I can additional check on this